internal/eval/eval.go

```diff
@@ -4108,6 +4110,91 @@ func evalPFMERGE(args []string, store *dstore.Store) []byte {
 	return clientio.RespOK
 }
 
+func evalPFDEBUG(args []string, store *dstore.Store) []byte {
+	if len(args) != 2 {
+		return diceerrors.NewErrArity("PFDEBUG")
+	}
+
+	subcommand := strings.ToUpper(args[0])
+	key := args[1]
+
+	obj := store.Get(key)
+	if obj == nil {
+		return diceerrors.NewErrWithMessage("Key does not exist")
```

Comment:
This should match the message returned by redis, and ideally be defined as a constant string in the constants file.

Reply:
yeah, matches the message. Will define as constant